mapiproxy/servers/default/nspi/emsabp.c

```diff
@@ -863,7 +862,9 @@ _PUBLIC_ enum MAPISTATUS emsabp_table_fetch_attrs(TALLOC_CTX *mem_ctx, struct em
 				lpProps.value.l = 0x0;
 				break;
 			case PidTagDisplayName:
-				lpProps.value.lpszW = NULL;
+				/* This value is temporal to workaround PropertyRow_addprop internals */
+				/* It will be set to NULL after the call to PropertyRow_addprop */
+				lpProps.value.lpszW = "t";
```

Comment:
This is because we pass a NULL to  PropertyRow_addprop the property is converted to an error property. 
The other way would we to fix PropertyRow_addprop to pass it null but this should be done as other issue. But I am not sure that the semantics of that functions allows to pass NULL values like that.